packages/help-center/src/components/help-center-article.scss

```
@@ -111,7 +111,7 @@
 				}
 			}
 
-			p {
+			p, ol {
```

```suggestion
			p, ol:not(.toc-parent-list) {
```

Actually can we avoid editing the table of contents list.

---

Good catch. In that case, we could restrict with `wp-block-list`, so there is no need to negate for new ordered lists. What do you think?

---

Noice. I love it.

---

Just pushed with this `wp-block-list` change, but let me know in case there is something else missing.